src/data/french-bank-holidays.js

```diff
@@ -0,0 +1,103 @@
+/*jshint esnext: true */
+
+(function(exports) {
+'use strict';
+
+function _easter (year, julian, gregorian) {
+    let k = Math.floor(year / 100)
+    let m = 15 + Math.floor((3 * k + 3) / 4) - Math.floor((8 * k + 13) / 25)
+    let s = 2 - Math.floor((3 * k + 3) / 4)
+    if (julian) {
+      m = 15
+      s = 0
+    }
+    let a = year % 19
+    let d = (19 * a + m) % 30
+    let r = Math.floor((d + a / 11) / 29)
+    let og = 21 + d - r
+    let sz = 7 - Math.floor(year + year / 4 + s) % 7
+    let oe = 7 - (og - sz) % 7
+    let os = og + oe
+    if (gregorian) {
+      os = os + Math.floor(year / 100) - Math.floor(year / 400) - 2
+    }
+    //                      1   2   3   4   5   6   7   8
+    let daysPerMonth = [0, 31, 28, 31, 30, 31, 30, 31, 31]
+    let day = os
+    let month
+    for (month = 3; month < 8; month++) {
+      if (day <= daysPerMonth[month]) {
+        break
+      }
+      day -= daysPerMonth[month]
+    }
+
+    return new Date(year, month - 1, day)
+}
+
+function sortHolidays(previousDay, currentDay) {
+  // Compare months
+  if (previousDay[0] < currentDay[0]) return -1;
+  if (previousDay[0] > currentDay[0]) return 1;
+  // Compare days
+  if (previousDay[1] < currentDay[1]) return -1;
+  if (previousDay[1] > currentDay[1]) return 1;
+  return 0;
+}
+
+function getFrenchBankHolidays(year) {
+  const FIXED_HOLIDAYS = [
+    [1, 1, 'New Years Day'],
+    [5, 1, "Labour Day"],
+    [5, 8, "Victory in Europe Day"],
+    [7, 14, "Bastille Day"],
+    [8, 15, "Assumption Day"],
+    [11, 1, "All Saints Day"],
+    [11, 11, "Armistice Day"],
+    [12, 25, "Christmas Day"],
```

Comment:
I'd do: `(date) => { date = new Date(date); date.setDate(date.getDate() + 1)); return date; }` because it's more readable, but n1k0's idea is more functional :) Both works for me as long as it's in its own separate function that makes it easy to read when we use it.